bootstrap/standalone/src/main/java/org/geysermc/platform/standalone/GeyserStandaloneBootstrap.java

```diff
@@ -108,6 +108,9 @@ public void onEnable() {
         try {
             File configFile = FileUtils.fileOrCopiedFromResource("config.yml", (x) -> x.replaceAll("generateduuid", UUID.randomUUID().toString()));
             geyserConfig = FileUtils.loadConfig(configFile, GeyserStandaloneConfiguration.class);
+            if (this.geyserConfig.getRemote().getAddress().equalsIgnoreCase("auto")) {
+            	geyserConfig.getRemote().setAddress("127.0.0.1");
```

Comment:
Indentation here looks off?

Reply:
That's my mistake, now i fixed it, thank you.